apis/components/v1alpha1/datasciencepipelines/types.go

```diff
@@ -0,0 +1,23 @@
+// +kubebuilder:object:generate=true
+
+// Package datasciencepipelines provides a set of types used for DataSciencePipelines component
+package datasciencepipelines
+
+import operatorv1 "github.com/openshift/api/operator/v1"
+
+type PreloadedPipelinesSpec struct {
+	// Configures whether to automatically import the InstructLab pipeline when new pipeline servers or DSPAs are created.
+	// You must enable the trainingoperator component to run the InstructLab pipeline.
```

Comment:
We're changing this to perform the import on existing pipelines now.
```suggestion
	// Configures whether to automatically import the InstructLab pipeline.
	// You must enable the trainingoperator component to run the InstructLab pipeline.
```